docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,757 @@
+* `transaction.name` 
+--
+
+anomalous_error_rate_for_user_agents_jsbase::
+Detects user agents that are encountering errors at an above normal rate. This 
+can help detect browser compatibility issues.
++
+--
+Bucket span: 15m.
+
+Function: `high_non_zero_count`.
+
+Influencers:
+
+* `user_agent.name`
+* `error.exception.message.keyword`
+* `error.page.url`
+* `service.name`
+--
+
+decreased_throughput_jsbase::
+decreased_throughput_nodejs::
+Identifies periods during which the application is processing fewer requests 
+than normal.
++
+--
+Bucket span: 15m.
+
+Function: `low_count`.
+
+Influencers:
+
+* `service.name`
+* `transaction.name`
+--
+
+high_count_by_user_agent_jsbase::
+Detects user agents that are making requests at a suspiciously high rate. This 
+is useful in identifying bots.
++
+--
+Bucket span: 15m.
+
+Function: `high_non_zero_count`.
+
+Influencers:
+
+* `service.name`
+* `user_agent.name`
+--
+
+high_mean_response_time::
+Detects anomalies in high mean of transaction duration.
++
+--
+Bucket span: 15m.
+
+Function: `high_mean`.
+--
+
+
+[float]
+[[ootb-ml-jobs-auditbeat]]
+=== Auditbeat
+
+docker_high_count_process_events_ecs::
+Detects unusual increases in process execution rates in Docker containers.
++
+--
+Bucket span: 1h.
+
+Function: `high_count`.
+
+Influencers:
+
+* `container.name`
+* `process.executable`
+--
+
+docker_rare_process_activity_ecs::
+Detects rare process executions in Docker containers.
++
+--
+Bucket span: 1h.
+
+Function: `rare`.
+
+Influencers:
+
+* `container.name`
+* `process.executable`
+--
+
+hosts_high_count_process_events_ecs::
+Detects unusual increases in process execution rates.
++
+--
+Bucket span: 1h.
+
+Function: `high_non_zero_count`.
+
+Influencers:
+
+* `host.name`
+* `process.executable`
+--
+
+hosts_rare_process_activity_ecs::
+Detects rare process executions on hosts.
++
+--
+Bucket span: 1h.
+
+Function: `rare`.
+
+Influencers:
+
+* `host.name`
+* `process.executable`
+--
+
+
+[float]
+[[ootb-ml-jobs-logs-ui]]
+=== Logs UI
+
+log_entry_categories_count::
+Detects anomalies in count of log entries by category.
++
+--
+Bucket span: 15m.
+
+Function: `count`.
+
+Influencers:
+
+* `event.dataset`
+* `mlcategory`
+--
+
+log_entry_rate::
+Detects anomalies in the log entry ingestion rate.
++
+--
+Bucket span: 15m.
+
+Function: `count`.
+
+Influencers:
+
+* `event.dataset`
+--
+
+[float]
+[[ootb-ml-jobs-metricbeat]]
+=== Metricbeat
+
+high_mean_cpu_iowait_ecs::
+Detects unusual increases in cpu time spent in iowait.
++
+--
+Bucket span: 10m.
+
+Function: `high_mean`.
+
+Influencers:
+
+* `host.name`
+--
+
+max_disk_utilization_ecs::
+Detects unusual increases in disk utilization.
++
+--
+Bucket span: 10m.
+
+Function: `max`.
+
+Influencers:
+
+* `host.name`
+--
+
+metricbeat_outages_ecs::
+Detects unusual decreases in Metricbeat documents.
++
+--
+Bucket span: 10m.
+
+Function: `low_count`.
+
+Influencers:
+
+* `event.dataset`
+--
+
+
+[float]
+[[ootb-ml-jobs-nginx]]
+=== Nginx
+
```

Comment:
Could we add something like this here too (moved out of https://www.elastic.co/guide/en/machine-learning/master/create-jobs.html)?:

> Context: These job wizards appear in Kibana if you use Filebeat to ship access logs from your Nginx HTTP servers to Elasticsearch and store it using fields and datatypes from the Elastic Common Schema (ECS).